### PR TITLE
bug 1659142 adding project parameters

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1659142

@openshift/team-documentation, will you PTAL?

@jianlinliu, will you please review and confirm that this bug is applicable to at least 3.6-3.11?

### CODE REVIEW

**Comment by wmengRH** [comment]
LGTM. 
It is good to apply from 3.6 to 3.11.

**Comment by kalexand-rh** [comment]
Thank you!

@openshift/team-documentation, will you PTAL? QE's approved the change, and I'd like to merge.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -791,6 +791,59 @@ You can use the default *node-config-master-infra* node group to achieve this:
...
+|Parameter |Description |Type |Default Value
+
+|`osm_project_request_message`
+|The string presented to a user if they are unable to request a project via the
+projectrequest API endpoint.
+|String
+| null
+
+|`osm_project_request_template`
+|The template to use for creating projects in response to projectrequest. If you
```
Comment (huffmanca) [inline install/configuring_inventory_file.adoc]: Should this be:

> ...in response to **a** projectrequest.

This is my ignorance with 3.x showing. :)

**Review by huffmanca — COMMENTED** [review]
One trivial comment. The rest looks good!

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install/configuring_inventory_file.adoc]: I think you're right. I'm fixing this in both instances. Thanks!

**Pull request merged**